public/lib/filter-defaults.js

```diff
@@ -125,29 +127,26 @@ var filterDefaults = function (statuses, wfFiltersService) {
             customLinkFunction: ['wfDateParser', '$scope', (wfDateParser, $scope) => {
 
                 $scope.dateOptions = wfDateParser.getDaysThisWeek();
-                const storedSelectedDate = wfFiltersService.get('selectedDate');
-                var selectedDate = storedSelectedDate ? storedSelectedDate : '';
+                const storedSelectedDate = wfFiltersService.get('deadline');
+                const deadline = storedSelectedDate ? storedSelectedDate : '';
 
-                // ensure that the date from the URL is the same object as the
-                // one used in the Select drop-down, as its compared with ===
-                $scope.dateOptions.forEach(function (date) {
-                    if (date.isSame(selectedDate)) {
-                        selectedDate = date;
-                        $scope.selectedFilters = ['customDate'];
-                    }
-                });
+                // Can our deadline be parsed into a date?
```

Comment:
```suggestion
                // Can our deadline be parsed into a date?
                // If so, it's a custom date filter (rather than 'today', 'tomorrow' etc)
```

Reply:
(Just to address why we care!)

Reply:
I most probably misunderstood, but we *do want* to keep relative dates as they are immensely useful. Eg a bookmark can always show _tomorrow_ whatever day it’s clicked on. Actually, we may need more relative date choices (but here, I defer to John S.).

Reply:
Today, tomorrow, and weekend still behave as normal 👍